site/app/config/legacy_aliases.yml

```diff
@@ -0,0 +1,2 @@
+services:
+        twig.extension.date: '@Twig\Extensions\DateExtension'
```

Comment:
That one should not be needed, we don't refer to the extension by name anywhere.